models/token_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type TokenTestSuite struct {
+	suite.Suite
+}
```

Comment:
I used test suites because they can ensure that `CreateTestEngine()` is called before any tests run. If there is some way to ensure this without suites, I'd be open to changing.